src/args.rs

```diff
@@ -235,7 +235,8 @@ pub struct CliArgs {
     #[arg(short = 't', long = "title", env = "MINISERVE_TITLE")]
     pub title: Option<String>,
 
-    /// Set custom header for responses
+    /// Inserts custom headers into the responses. Specify each header as a 'Header:Value' pair.
+    /// This parameter can be used multiple times to add multiple headers.
```

Comment:
Since you're already here:
1. Perhaps provide an example?
1. Maybe also document the behavior what happens when a header conflicts with a previous one?

Reply:
Thanks for the feedback, @svenstaro! I've updated the PR:

- Added an example in the README demonstrating how to use the `--header` option for enabling HSTS, which helps in achieving an A+ SSL Labs rating.
- Clarified the behavior when header conflicts occur: new headers will not overwrite existing values if already set.

Please let me know if further adjustments are needed.